impl/src/main/java/io/jsonwebtoken/impl/io/RuntimeClasspathDeserializerLocator.java

```diff
@@ -45,12 +46,10 @@
 
     @SuppressWarnings("WeakerAccess") //to allow testing override
     protected Deserializer<T> locate() {
-        if (isAvailable("io.jsonwebtoken.jackson.io.JacksonDeserializer")) {
-            return Classes.newInstance("io.jsonwebtoken.jackson.io.JacksonDeserializer");
-        } else if (isAvailable("io.jsonwebtoken.orgjson.io.OrgJsonDeserializer")) {
-            return Classes.newInstance("io.jsonwebtoken.orgjson.io.OrgJsonDeserializer");
-        } else if (isAvailable("io.jsonwebtoken.gson.io.GsonDeserializer")) {
-            return Classes.newInstance("io.jsonwebtoken.gson.io.GsonDeserializer");
+        ServiceLoader<Deserializer> serviceLoader = ServiceLoader.load(Deserializer.class);
```

Comment:
Can this entire class be replaced by a call to `Services.loadFirst` ?

Reply:
Yes! yes it can